src/api/cpp/cvc5_parser.cpp

```diff
@@ -345,23 +346,33 @@ void InputParser::setStringInputInternal(const std::string& input,
   d_parser->setStringInput(input, name);
 }
 
+void InputParser::setIncrementalStringInputInternal(modes::InputLanguage lang,
+                                                    const std::string& name)
+{
+  // initialize the parser
+  d_parser = Parser::mkParser(lang, d_solver, d_sm->toSymManager());
+  initializeInternal();
+  d_istringStream.str("");
+  d_istringStream.clear();
+  d_parser->setStreamInput(d_istringStream, name);
+}
```

Comment:
This is missing the usual `CVC5_API_TRY_CATCH_BEGIN` handling.

Reply:
This is an internal helper method, so this shouldn't be done, right?

Reply:
Ah right, in this case it's fine.